src/ucp/rma/flush.c

```diff
@@ -115,7 +118,7 @@ static void ucp_ep_flush_progress(ucp_request_t *req)
             }
         } else {
             ucp_ep_flush_error(req, status);
-            break;
+            req->send.flush.started_lanes |= UCS_BIT(lane);
```

Comment:
should we do
`--req->send.state.uct_comp.count;`?

Reply:
move `req->send.flush.started_lanes |= UCS_BIT(lane);` to `ucp_ep_flush_error()`?

Reply:
> should we do
> `--req->send.state.uct_comp.count;`?

it's done in `ucp_ep_flush_error`

Reply:
> move `req->send.flush.started_lanes |= UCS_BIT(lane);` to `ucp_ep_flush_error()`?

it will be wrong for `pending_add` failure